prebuild_binaries/linux_build_all.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Fail on any error.
+set -e pipefail
+
+# Display commands
+set -x
+
+case $KOKORO_JOB_TYPE in
```

Comment:
Should this file go to .kokoro directory?

Reply:
.kokoro is where the configs that are used for like the unit tests. We'll get automated PRs there. I think we want to keep these separate for now. 

But maybe we should have the prebuild_binaries (which builds the binaries) and the testing (which runs the e2e tests) the directories together. Moving any of the currently used configs, though, could get a bit tricky (because we also have to modify the google3 kokoro configs at the same time), so we'd have to delete the configs in "testing" in a separate PR.

I'll think more on this.